### PR TITLE
removed extra endif()

### PR DESCRIPTION
**Pull request by kab163**

### CODE REVIEW

**Comment by kab163** [comment]
LGTM

File: CMakeLists.txt [hunk 1]
```diff
@@ -60,7 +60,6 @@ set(CMAKE_CUDA_STANDARD 11)
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "PGI")
     message(STATUS "Setting CMAKE_CXX_EXTENSIONS to ON for PGI Compiler")
```
Comment (davidbeckingsale) [inline CMakeLists.txt:61]: Can you please change this so it's only indented 2 spaces to match the rest of the file.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 ## [Develop Branch]
 
+### Removed
+
+- Removed extraneous 'endif()' line in Umpire CMakeLists.txt file
```
Comment (davidbeckingsale) [inline CHANGELOG.md:13]: I think this should be in "Fixed" rather than "Removed".

**Review by davidbeckingsale — CHANGES_REQUESTED** [review]
Super quick changes.

File: CHANGELOG.md (hunk 2 above)
Comment (kab163) [reply CHANGELOG.md:13]: Sure thing

File: CMakeLists.txt (hunk 1 above)
Comment (kab163) [reply CMakeLists.txt]: Done!

**Comment by kab163** [comment]
LGTM

**Review by davidbeckingsale — APPROVED** [review]

**Pull request merged**